pkg/bindings/manifests/manifests.go

```diff
@@ -6,7 +6,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"io/ioutil"
+	"io"
```

Comment:
Doubled "io" import - can just get rid of this

Reply:
Thanks, ya, I'm sure there's a bunch of problems.  I'm just letting the CI system find them one by one while I listen in on our meeting :grin: